source/simulator/core.cc

```diff
@@ -1481,6 +1484,11 @@ namespace aspect
     TimerOutput::Scope timer (computing_timer, "Postprocessing");
     pcout << "   Postprocessing:" << std::endl;
 
+    pcout << "   Total wallclock time elapsed since absolute start:" << walltime.wall_time()+walltime_from_resume << std::endl;
```

Comment:
I don't think outputting this every timestep is something we should do. Also: 
```suggestion
    pcout << "   Total wallclock time including restarts:" << walltime.wall_time()+walltime_from_resume << std::endl;
```

Reply:
I agree I moved it into source/simulator/checkpoint_restart.cc